content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+
+For example, if you named the cluster "edf-cluster-a.mycompany.com" when you installed the HPE Ezmeral Data Fabric cluster, then you can use the following keytool command:
+
+```shell
+${JAVA_HOME}/bin/keytool -noprompt -importcert -file /opt/mapr/conf/ca/chain-ca.pem -alias edf-clustera-ca -keystore ${JAVA_HOME}/lib/security/cacerts - storepass <cacerts_truststore>
+```
+
+The file - /opt/mapr/conf/ca/chain-ca.pem, is a self-signed TLS certificate file created by the Installer when configuring wire-level encryption for the cluster.
+
+Since it's a self-signed TLS certificate, the client (application, or your browser) will not be able to trust the TLS certificate of the server when accessing the HPE Ezmeral Data Fabric server.
+
+This is because the CA certificate used by the self-signed TLS certificate is not publicly trusted.
+
+In any scenario where a self-signed TLS certificate is used, you need to import the self-signed CA certificate into the OS system.
+
+You can use the following command:
+
+```shell
+keytool -list -v -cacerts
+```
+
+to look at things like CA certificates in your JVM.
+
+You'll find something like the following:
+
+```dart
+Alias name: digicertassuredidrootca
+Creation date: Feb 2, 2023
+Entry type: trustedCertEntry
+
+Owner: CN=DigiCert Assured ID Root CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+Issuer: CN=DigiCert Assured ID Root CA, OU=www.digicert.com, O=DigiCert Inc, C=US
+...
+```
+
+This is a recognized CA certificate issued by Digicert.
+
+So, after you use `keytool` to import /opt/mapr/conf/ca/chain-ca.pem into the JVM, use keytool -list -v -cacerts to see something similar to the following:
+
+```dart
+Alias name: edf-clustera-ca
+Creation date: Feb 16, 2023
+Entry type: trustedCertEntry
+
+Owner: CN=MapR Engineering Signing CA, OU=MapR Engineering Signing CA, O=MapR, DC=hpecorp, DC=net
+Issuer: CN=MapR Engineering Root CA, OU=MapR Engineering Root CA, O=MapR, DC=hpecorp, DC=net
+...
+```
+
+#### 2. The documentation describes several client software that you can use to interact with the HPE Ezmeral Object Store. I would like to add the following clarification:
+
+When using the `mc` command-line tool or the `s3cmd` command-line tool to interact with HPE Ezmeral Data Fabric Object Store without completing the configuration of "[Enabling S3 Virtual-Host-Style Requests](https://docs.datafabric.hpe.com/72/AdvancedInstallation/Enabling_object_store.html#concept_isb_53h_5bb__section_nnp_kr2_bvb)", some commands will not work properly.
+
+In this case, for management operations, such as creating accounts, IAM accounts, buckets, etc., I recommend you use the `mc` command line tool.
+
+For object listing, getting, putting, and deleting operations, I recommend you use `AWS CLI`.
+
+#### 3. Cannot revert to HTTP mode after enabling HTTPS.
+
+If the Object Store was installed using the Installer, the Object Store would also have security enabled, including HTTPS.
+
+Although you can see the below description:
```

Comment:
Perhaps this might be clearer?
"For example, although you can see the description below from HTTP Access to Object Store, this configuration change alone does not change Object Store to HTTP mode.

        To revert to http access, comment out the moss.certs.dir=/opt/mapr/conf line...

Additionally, there is nothing else in HPE Ezmeral Data Fabric's documentation on how to modify Object Store's TLS mode.

[so... is there anything they can do about it? What's your recommendation to them for this? Even if you say "Don't know what to tell you about this." it would provide an ending to the thought.